include/RED4ext/RTTITypes-inl.hpp

```diff
@@ -326,7 +326,7 @@ RED4EXT_INLINE bool RED4ext::CClass::sub_D0() const
     return func(this);
 }
 
-RED4EXT_INLINE RED4ext::ScriptInstance RED4ext::CClass::AllocInstance(bool aZeroMemory) const
+RED4EXT_INLINE RED4ext::ScriptInstance RED4ext::CClass::CreateInstance(bool aZeroMemory) const
 {
     using func_t = ScriptInstance (*)(const CClass*, uint32_t, bool);
     RelocFunc<func_t> func(Addresses::CClass_AllocInstance);
```

Comment:
Please update this address to be `CreateInstance`.

Reply:
Done.